pkg/scheduler/plugins/elasticquota/plugin.go

```diff
@@ -229,7 +229,7 @@ func (g *Plugin) PreFilter(ctx context.Context, cycleState *framework.CycleState
 	podRequest, _ := core.PodRequestsAndLimits(pod)
 	used := quotav1.Add(podRequest, state.used)
 
-	if isLessEqual, exceedDimensions := quotav1.LessThanOrEqual(used, state.runtime); !isLessEqual {
+	if isLessEqual, exceedDimensions := lessThanOrEqual(used, state.runtime, podRequest); !isLessEqual {
```

Comment:
I think there is no need to change this place. Just make sure that the resource dimension used is equal to pod request.  
That is, after line 230, change it to the following:
```go
used := quotav1.Add(podRequest, state.used)
used = quotav1.Mask(used, quotav1.ResourceNames(podRequest))
```
A more thorough modification is to find the intersection of used and request in the snapshotPostFilterState method.